mappings/net/minecraft/entity/ai/brain/Brain.mapping

```diff
@@ -0,0 +1,22 @@
+CLASS aiu net/minecraft/entity/ai/brain/Brain
+	FIELD a memories Ljava/util/Map;
+	FIELD b sensors Ljava/util/Map;
+	FIELD c tasks Ljava/util/Map;
+	FIELD d schedule Lavy;
+	FIELD g activities Ljava/util/Set;
```

Comment:
Iirc these are all the activities that can currently be performed -- maybe this name should be more specific?